src/test/ui/nll/closure-requirements/propagate-approximated-to-empty.rs

```diff
@@ -0,0 +1,47 @@
+// Copyright 2016 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// Test a case where we fail to approximate one of the regions and
```

Comment:
Similarly to `escape-argument-callee`, a test case where the closure requires a relationship between 2 unrelated higher-ranked regions, with no helpful relations between the HBRs and free regions.

Maybe also add to this test cases where:
```
'c: 'x, 'b: 'y
\- CAN'T HELP, because we can't prevent 'x and 'y from being assigned "different" short lifetimes

'y: 'c
\- can't help, because we can't prevent 'x from being short.
```